kafka_utils/util/offsets.py

```diff
@@ -546,7 +546,7 @@ def set_consumer_offsets(
 def _nullify_partition_offsets(partition_offsets):
     result = {}
     for partition in partition_offsets:
-        result[partition] = -1
+        result[partition] = 0
```

Comment:
We can probably add a comment here, perhaps also a link to the upstream Kafka code.